### PR TITLE
Fix the input object example

### PR DESCRIPTION
**Pull request by harendra-kumar**

### CODE REVIEW

File: examples/InputObject.hs [hunk 1]
```diff
@@ -48,7 +48,7 @@ example = interpretAnonymousQuery @Query root
 
 main :: IO ()
 main = do
-  response <- example "{ description(dogStuff: {toy: \"bone\", likesTreats: true}) }"
+  response <- example "{ description(dogStuff: {_toy: \"bone\", _likesTreats: true}) }"
```
Comment (teh) [inline examples/InputObject.hs]: Thanks for the PR! 

I'm not entirely sure what's going on here. Is there a bug for reference? Can you maybe add a description to the commit message for why the example didn't work before and why it works with this fix?

Comment (jml) [reply examples/InputObject.hs]: What breaks because of this?

**Comment by harendra-kumar** [comment]
It generates this output without the change:

```
{"data":{"description":null},"errors":[{"message":"Could not coerce Name {unName = \"dogStuff\"} to valid value: Key not found: Name {unName = \"_toy\"}"}]}
{"data":{"description":"their favorite toy is a shoe"}}
```

With the change:

```
{"data":{"description":"likes treats and their favorite toy is a bone"}}
{"data":{"description":"their favorite toy is a shoe"}}
```

**Comment by jml** [comment]
We had both approved this, and there were no outstanding comments, so I merged it.

**Pull request merged**